apps/etterlatte-vedtaksvurdering/src/test/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtaksvurderingRouteTest.kt

```diff
@@ -344,14 +348,21 @@ internal class VedtaksvurderingRouteTest {
                         header(HttpHeaders.Authorization, "Bearer $token")
                     }.let {
                         it.status shouldBe HttpStatusCode.OK
-                        deserialize<VedtakSammendragDto>(it.bodyAsText())
+                        objectMapper.readValue<VedtakSammendragDto>(it.bodyAsText())
                     }
 
             vedtaksammendrag.id shouldBe attestertVedtak.id.toString()
             vedtaksammendrag.behandlingId shouldBe attestertVedtak.behandlingId
             vedtaksammendrag.datoAttestert shouldNotBe null
-            // TODO datoAttestert leses inn som UTC selv om den er +01:00 - ville forventet med norsk tidssone?
-            // vedtaksammendrag.datoAttestert shouldBe attestertVedtak.attestasjon?.tidspunkt?.toNorskTid()
+            /*
+            ISO_OFFSET_DATE_TIME brukes internt i jackson for https://github.com/FasterXML/jackson-modules-java8/tree/master/datetime
+            mens toNorskTid "bruker" ISO_ZONED_DATE_TIME
+             */
+            vedtaksammendrag.datoAttestert?.format(ISO_OFFSET_DATE_TIME) shouldBe
+                attestertVedtak.attestasjon
+                    ?.tidspunkt
+                    ?.toNorskTid()
+                    ?.format(ISO_OFFSET_DATE_TIME)
```

Comment:
Dette burde kanskje bli default for de som bruker norsktid

Reply:
Eller bare legge til egen serializer/deserializer:
```
object ZonedDateTimeSerializer : JsonSerializer<ZonedDateTime?>() {
    override fun serialize(value: ZonedDateTime?, gen: JsonGenerator?, serializers: SerializerProvider?) {
        gen?.writeString(DateTimeFormatter.ISO_ZONED_DATE_TIME.format(value));
        return;
    }
}
```

Reply:
Men det er ikke ISO compliant så usikker på om det er riktig https://github.com/FasterXML/jackson-modules-java8/issues/170

Reply:
Kjens som vi ikkje heilt har tatt valet om korvidt vi skal bruke norsk tid eller UTC, så vi er litt inkonsekvente rundt omkring i kodebasen. Hadde vore fint å lande på det eine eller andre.